src/routes/settings/assistants/new/+page.server.ts

```diff
@@ -18,6 +21,10 @@ const newAsssistantSchema = z.object({
 	exampleInput3: z.string().optional(),
 	exampleInput4: z.string().optional(),
 	avatar: z.instanceof(File).optional(),
+	generateAvatar: z
```

Comment:
do those fields have to be mutually exclusive or not? Basically, both of them can't be active right?

Reply:
Yeah pretty much. I do the check a bit later down but do you think there's a nice way to do it with zod directly?

Reply:
> I do the check a bit later

oh I see & that's what I meant as well.

> do you think there's a nice way to do it with zod directly

I don't think so